pkg/provider/cloud/kubevirt/provider.go

```diff
@@ -177,3 +184,22 @@ func GetCredentialsForCluster(cloud kubermaticv1.CloudSpec, secretKeySelector pr
 
 	return kubeconfig, nil
 }
+
+func (k *kubevirt) updateInfraStorageClassesInfo(ctx context.Context, spec *kubermaticv1.CloudSpec) error {
```

Comment:
There is no need to bind function `updateInfraStorageClassesInfo` to the `kubevirt` type. Please move the function to the `storage.go` file.